runtime/common/xwalk_content_client.h

```diff
@@ -9,6 +9,7 @@
 #include <vector>
 
 #include "base/compiler_specific.h"
+#include "base/memory/ref_counted_memory.h"
```

Comment:
Use forward declaration for RefCountedStaticMemory instead of #include here. Refer to Chromium C++ guidelines.